install/updates/50-krbenctypes.update

```diff
@@ -1,8 +1,8 @@
 dn: cn=$REALM,cn=kerberos,$SUFFIX
-add: krbSupportedEncSaltTypes: camellia128-cts-cmac:normal
-add: krbSupportedEncSaltTypes: camellia128-cts-cmac:special
-add: krbSupportedEncSaltTypes: camellia256-cts-cmac:normal
-add: krbSupportedEncSaltTypes: camellia256-cts-cmac:special
+${FIPS}add: krbSupportedEncSaltTypes: camellia128-cts-cmac:normal
+${FIPS}add: krbSupportedEncSaltTypes: camellia128-cts-cmac:special
+${FIPS}add: krbSupportedEncSaltTypes: camellia256-cts-cmac:normal
+${FIPS}add: krbSupportedEncSaltTypes: camellia256-cts-cmac:special
 add: krbSupportedEncSaltTypes: aes128-sha2:normal
```

Comment:
This isn't going to work. It doesn't need the {} wrapper and either way it is going to fail because this translates into # add and the skip for leading comments has already passed so it tries to evaluate the command '# add' which is unknown so the update fails. Additional changes in the updater will be necessary.